express-delete/index.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', function (req, res) {
+  const gradesArray = [];
+  for (const key in grades) {
+    gradesArray.push(grades[key]);
+  }
+  res.json(gradesArray);
+});
+
+app.delete('/api/grades/:id', function (req, res) {
+  delete grades[req.params.id];
+  res.sendStatus(404);
```

Comment:
Should be a status of `204` here, which means `'No Content'`. A `404` code means `'Not Found'`